docs/Getting Started/FreeBSD.rst

```diff
@@ -0,0 +1,129 @@
+FreeBSD
+=======
+
+|ZoF-logo|
+
+Installation on FreeBSD
+-----------------------
+
+OpenZFS is available in the FreeBSD ports tree as sysutils/openzfs and
```

Comment:
We should clarify that OpenZFS is also in the FreeBSD base system now on FreeBSD 13.0-CURRENT and later.

Reply:
Done